airbyte-integrations/base-python/airbyte_protocol/models/airbyte_catalog.py

```diff
@@ -0,0 +1,19 @@
+# generated by datamodel-codegen:
+#   filename:  airbyte_catalog.yaml
+
+from __future__ import annotations
```

Comment:
why are we commiting this to source and not committing the java types?

Reply:
From the slack thread it seems like we'll do this for now.